tests/conversion_test.py

```diff
@@ -152,6 +155,221 @@ def test_pytket_qir_7() -> None:
     check_qir_result(result, "test_pytket_qir_7")
 
 
+def test_pytket_qir_8() -> None:
+    # test setbits op
+    c = Circuit(1, name="test_classical")
+    a = c.add_c_register("a", 8)
+
+    c.add_c_setbits([True], [a[0]])
+    c.add_c_setbits([True], [a[2]])
+    c.add_c_setbits([True], [a[1]])
+    c.add_c_setbits([True], [a[7]])
+    c.add_c_setbits([False, True] + [False] * 6, list(a))
+
+    assert c.n_qubits == 1
+    assert c.n_bits == 8
+
+    result = pytket_to_qir(c, name="test_pytket_qir_8", qir_format=QIRFormat.STRING)
+
+    check_qir_result(result, "test_pytket_qir_8")
+
+
+def test_pytket_qir_9() -> None:
+    # test copybits op
+    c = Circuit(1, name="test_classical")
+    a = c.add_c_register("a", 2)
+    b = c.add_c_register("b", 2)
+
+    c.add_c_copyreg(a, b)
+
+    assert c.n_qubits == 1
+    assert c.n_bits == 4
```

Comment:
Small point, but I don't think these asserts (and similar ones below) are relevant here; they're not testing pytket-qir at all.

Reply:
I have cleaned that up, done in @cqc-melf
[remove qubit and bit asserts](https://github.com/CQCL/pytket-qir/pull/45/commits/ace002f6e8fceb14e15e00e30e1874296fc3eb2b)
[ace002f](https://github.com/CQCL/pytket-qir/pull/45/commits/ace002f6e8fceb14e15e00e30e1874296fc3eb2b)